src/main.rs

```diff
@@ -79,17 +80,21 @@ fn run() -> Result<(), ContextualError> {
         TermLogger::init(LevelFilter::Error, Config::default())
     };
 
-    if miniserve_config.no_symlinks
-        && miniserve_config
+    if miniserve_config.no_symlinks {
+        let is_symlink = miniserve_config
```

Comment:
Clippy was complaining about the "complex condition being evaluated in the `if` block", so I split it out and CI now passes.

Reply:
That's fine. I generally prefer formatting and refactoring stuff like this in a separate PR, though.

Reply:
Yeah, I would have skipped it if CI hadn't complained.